ena-submission/src/ena_deposition/ena_submission_helper.py

```diff
@@ -191,11 +161,11 @@ def create_ena_project(config: ENAConfig, project_set: ProjectSet) -> CreationRe
     errors = []
     warnings: list[str] = []
 
+    xml = get_project_xml(project_set)
```

Comment:
we might also want to wrap this with a try catch

Reply:
Why? We already have too much try/except and there's no network requests here - if it blows up it blows up?